pynetdicom/tests/hide_modules.py

```diff
@@ -26,14 +26,10 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-try:
-    import importlib.abc
-
-    # py>=3.3 has MetaPathFinder
-    _ModuleHiderBase = getattr(importlib.abc, "MetaPathFinder")
-except ImportError:
-    # py2
-    _ModuleHiderBase = object
+import importlib.abc
+
+# py>=3.3 has MetaPathFinder
+_ModuleHiderBase = getattr(importlib.abc, "MetaPathFinder")
```

Comment:
Why still use `getattr`? Actually, this variable should not be needed anymore, just use 
```py
from importlib.abc import MetaPathFinder

class class ModuleHider(MetaPathFinder):
    ...
```